smsdk/_version.py

```diff
@@ -0,0 +1,113 @@
+#
+"""
+Track version information for sightmachine-sdk module here.
+"""
+import typing
+import itertools
+import requests
+import warnings
+import functools
+import re
+
+
+class VersionInfo(typing.NamedTuple):
+    major: typing.Any
+    minor: typing.Any
+    patchlevel: typing.Any
+    releaselevel: typing.Any
+    serial: typing.Any
+
+    def __str__(self) -> str:
+        result = "".join(
+            itertools.chain(
+                ".".join(str(i) for i in self[:3] if i is not None),
+                (str(i) for i in self[3:] if i is not None),
+            )
+        )
+        return result
+
+
+version_info = VersionInfo(1, 26, 0, "", None)
+version = str(f"v{version_info}")
+
+
+def sort_releases_descending(
+    releases: typing.List[typing.Dict[str, typing.Any]]
+) -> typing.List[typing.Dict[str, typing.Any]]:
+    def version_key(
+        release: typing.Dict[str, typing.Any]
+    ) -> typing.Union[typing.Tuple[int, int, int, str], typing.Tuple[()]]:
+        version = release["tag_name"]
+        # Use regular expression to extract numerical and string parts
+        match = re.match(r"(?i)v(\d+)\.(\d+)\.(\d+)(\D*)", version)
+        if match:
+            major, minor, patch, suffix = map(match.group, (1, 2, 3, 4))
+            return int(major), int(minor), int(patch), suffix
+        else:
+            return ()
+
+    return (
+        sorted(releases, key=version_key, reverse=True)
+        if len(releases) > 1
+        else releases
+    )
+
+
+def get_latest_release_version(
+    releases: typing.List[typing.Dict[str, typing.Any]]
+) -> typing.Optional[typing.Any]:
```

Comment:
Should the return value be a string vs Any?
```suggestion
) -> t_.Optional[str]:
```